rosbag2_storage/CMakeLists.txt

```diff
@@ -40,12 +40,12 @@ target_include_directories(${PROJECT_NAME}
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/${PROJECT_NAME}>
 )
-ament_target_dependencies(
-  ${PROJECT_NAME}
-  pluginlib
-  rcpputils
-  rcutils
-  yaml_cpp_vendor)
+target_link_libraries(${PROJECT_NAME}
+  pluginlib::pluginlib
+  rcpputils::rcpputils
+  rcutils::rcutils
+  ${YAML_CPP_LIBRARIES}
```

Comment:
yaml-cpp::yaml-cpp

Reply:
@MichaelOrlov @clalancette `yaml-cpp` is exported as `yaml-cpp`, not `yaml-cpp::yaml-cpp` , so I replaced them with `yaml-cpp`.